force-app/main/default/objects/Contact/validationRules/Only_accept_number_in_mobile_fields.validationRule-meta.xml

```diff
@@ -2,7 +2,8 @@
 <ValidationRule xmlns="http://soap.sforce.com/2006/04/metadata">
     <fullName>Only_accept_number_in_mobile_fields</fullName>
     <active>true</active>
-    <errorConditionFormula>NOT(REGEX(MobilePhone, &quot;\\d*?(\\d*?){10}&quot;))</errorConditionFormula>
+    <errorConditionFormula>NOT(REGEX(MobilePhone, &quot;\\d*?(\\d*?){10}&quot;))
```

Comment:
Ulik formel på valideringsregel for denne og og phone. Skal det være slik?

Reply:
Endret